router/image/templates/nginx.conf

```diff
@@ -15,43 +15,47 @@ http {
     sendfile on;
     tcp_nopush on;
     tcp_nodelay on;
-
     # The Timeout value must be greater than the front facing load balancers timeout value.
     # Default is the deis recommended timeout value for ELB - 1200 seconds + 100s extra.
-    {{ $defaultTimeout := or (.deis_router_defaultTimeout) "1300" }}
```

Comment:
we use this value in a few places so we should only have the `if exists` logic in one place as we once did. Makes it easier to make changes in the future :)